cypress/integration/basic-functionalities/local-tests/reporter/2_Edit_report.spec.js

```diff
@@ -1,36 +1,47 @@
-      cy.visit(baseURL + "/list"); // Visiting main page of Reporter
-    });
-  });
-  
-  context("Test for editing a report", () => {
-    it("should be able to edit the name", () => {
-      cy.get("table > tbody > :first-child() > :last-child() > a:nth-child(2)").click(); // We click on the Edit button on the created report
-      cy.get('[data-test-id="input-name"]').clear().type("Edited cypress report"); // We edit the name of the report
-      cy.get('[data-test-id="button-save"]').click(); // We click on the Save button
-      cy.get('.b-toast-success'); // We check if the success toast appears
-    });
-    it("should be able to edit the handle", () => {
-      cy.get('[data-test-id="input-handle"]').clear().type(consts.cypressUpdate); // We edit the handle of the report
-      cy.get('[data-test-id="button-save"]').click(); // We click on the Save button
-      cy.get('.b-toast-success'); // We check if the success toast appears
-    });
-    it("should be able to edit the description", () => {
-      cy.get('[data-test-id="input-description"]').clear().type("This is an edited automated description."); // We edit the description
-      cy.get('[data-test-id="button-save"]').click(); // We click on the Save button
-      cy.get('.b-toast-success'); // We check if the success toast appears
-      cy.get('[data-test-id="button-back"]').click(); // We click on the Back button to go on the main page
-    });
-  });
-});
+describe('Test for editing a report', () => {
+  context('Test for logging in the user', () => {
+    it('should be able to log in successfully', () => {
+      cy.visit(baseURL + '/') // Localhost in the env file should be changed to reflect the Reporter host. We are visiting the main authentication page here.
+      cy.get('[data-test-id="input-email"]').type('cypress@test.com')
+      cy.get('[data-test-id="input-password"]').type('cypress123')
+      cy.get('[data-test-id="button-login-and-remember"]').click()
+      cy.visit(baseURL + '/list') // Visiting main page of Reporter
+    })
+  })
+
+  context('Test for editing a report', () => {
+    it('should be able to edit the name', () => {
+      cy.get('table > tbody > :first-child() > :last-child() > a:nth-child(2)').click() // We click on the Edit button on the created report
+      cy.get('[data-test-id="input-name"]').clear().type('Edited cypress report')
+      cy.get('[data-test-id="button-save"]').click()
+      cy.get('.b-toast-success') // We check if the success toast appears
+    })
+    it('should be able to edit the handle', () => {
+      cy.get('[data-test-id="input-handle"]').clear().type('cypress_handle_edited')
+      cy.get('[data-test-id="button-save"]').click()
+      cy.get('.b-toast-success') // We check if the success toast appears
+    })
+    it('should be able to edit the description', () => {
+      cy.get('[data-test-id="input-description"]').clear().type('This is an edited automated description.')
```

Comment:
Let's no split these 3 checks into separate it()s; it's more time and code consuming. Create a test in which you edit the fields and then check if the values are okay